Makefile

```diff
@@ -3,7 +3,7 @@ PWD := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 PROJECT=connectapi
 NETWORK=${PROJECT}_default
-RSC_VERSION=2022.09.0
+CONNECT_VERSION=2022.09.0
```

Comment:
I know you might not be able to fix all of the type checking to get to this point, but a TODO that we should pull this forward when we can would be good IMHO

Reply:
Done.